dashboard/src/actions/operators.ts

```diff
@@ -321,7 +321,7 @@ export function getResources(
         try {
           const csvResources = await Operators.listResources(
             cluster,
-            namespace,
+            csv.metadata.namespace,
```

Comment:
So this is why the namespace wasn't being set? Interesting

Reply:
Exactly, when having selected "all namespaces" in the switch button, `namespace` value would be `""`.